yourgov/components/FormMobileFoodVendorPermit/task/FormTaskFormState.ts

```diff
@@ -169,14 +169,44 @@ export const taskStep6FormSchema = z.object({
 
 export type TaskStep6FormSchema = z.infer<typeof taskStep6FormSchema>;
 
+export const taskStep7FormSchema = z.object({
```

Comment:
These are copy/paste from step 6 and will throw TS errors when update the individual page forms. At that point we can update these.